src/components/Data/Query.js

```diff
@@ -0,0 +1,81 @@
+import gql from "graphql-tag";
```

Comment:
i don't know any graph ql but this looks cool :)

Reply:
It's neat. Likely will be used frequently in data fetching because many projects index their data with protocols like The Graph, which is full of gql